test/hydration-test.js

```diff
@@ -15,7 +15,15 @@ describe('Hydration',()=>{
 
     })
     it('Should return fluid ounces of water consumed each day over the course of a week',()=>{
-        expect(hydration.returnUserWeeklyFluidConsumption(12,"2019/06/16")).to.deep.equal([56,88,29,46,31,20,76])
-        
+        expect(hydration.returnUserWeeklyFluidConsumption(1,"2019/06/15")).to.deep.equal(
+          [
+        {date: "2019/06/21", numOunces: 50},
+        {date: "2019/06/20", numOunces: 50},
+        {date: "2019/06/19", numOunces: 91},
+        {date: "2019/06/18", numOunces: 61},
+        {date: "2019/06/17", numOunces: 96},
+        {date: "2019/06/16", numOunces: 69},
+        {date: "2019/06/15", numOunces: 37},
+        ])
```

Comment:
This was a great refactor and will make it easier to ensure that we are linking the correct dates to numOunces for a user. Great call on that one.